### PR TITLE
Adds a runechat plane to fix the issues related to the chat messages

### PR DESCRIPTION
**Pull request by Azarak**
## About The Pull Request
Previously your chat messages would be obstructed by the blackness of blocked turfs, aswell as dark lighting and some other effects. This fixes that
It still has backdrop ambient occlusion like it used to.
dreamseeker_2020-10-21_13-03-59
dreamseeker_2020-10-21_13-04-18

## Why It's Good For The Game
Hearing shouldn't be affected by vision. This makes reading much easier on the eyes, aswell as playing a blind character.

## Changelog
:cl:
fix: Runechat messages now won't be obstructed by darkness or lighting, or other effects
/:cl:

### CODE REVIEW

File: code/_onclick/hud/plane_master.dm [hunk 1]
```diff
@@ -153,3 +153,14 @@
...
+
+/obj/screen/plane_master/runechat
+	name = "runechat plane master"
+	plane = RUNECHAT_PLANE
+	appearance_flags = PLANE_MASTER
+	blend_mode = BLEND_OVERLAY
+
+/obj/screen/plane_master/runechat/backdrop(mob/mymob)
+	filters = list()
+	if(istype(mymob) && mymob.client && mymob.client.prefs && mymob.client.prefs.ambientocclusion)
```
Comment (Rohesie) [inline code/_onclick/hud/plane_master.dm:165]:
```suggestion
	if(istype(mymob) && mymob?.client?.prefs?.ambientocclusion)
```

**Comment by Farquaar** [comment]
I dig it.

**Review by Rohesie — APPROVED** [review]

**Comment by bobbahbrown** [comment]
can you check if this has problems with virtual speakers (like radio messages)

aside from that it was mostly a choice i made to avoid revealing people's locations in dark environments, but maybe this is the preferred alternative

sincerely,
bobbah 'bee' brown

**Comment by Azarak** [comment]
> 
> 
> can you check if this has problems with virtual speakers (like radio messages)
> 
> aside from that it was mostly a choice i made to avoid revealing people's locations in dark environments, but maybe this is the preferred alternative
> 
> sincerely,
> bobbah 'bee' brown

Seems to work good!

**Pull request merged**